src/Sentry/BindableSentryOptions.cs

```diff
@@ -53,6 +53,15 @@ internal partial class BindableSentryOptions
     public bool? EnableSpotlight { get; set; }
     public string? SpotlightUrl { get; set; }
 
+    [Experimental(Infrastructure.DiagnosticId.ExperimentalFeature)]
+    public BindableSentryExperimentalOptions Experimental { get; set; } = new();
```

Comment:
```suggestion
    internal BindableSentryExperimentalOptions Experimental { get; set; } = new();
```

Reply:
accepted ... but reverted:
This change fails `BindableSentryOptionsTests`, as they do reflection over public properties only.